packages/web/src/pages/_app.tsx

```diff
@@ -9,6 +10,17 @@ class NextApp extends App<AppInitialProps & WithApolloProps<{}>> {
     super.componentDidCatch(error, errorInfo)
   }
 
+  componentDidMount = () => {
+    const { ethereum } = window
+    if (!ethereum) {
+      // TODO: fix design and messages.
+      Modal.error({
+        title: 'This is an error message',
+        content: 'some messages...some messages...'
+      })
```

Comment:
@aggre 
Since I'm putting in a temporary message, could you just rewrite the text here?